examples/oval.rb

```diff
@@ -10,17 +10,18 @@
 ) do
   flow do
     para "Positional arguments:"
-    oval 30, 30, 80, 200, center: true
-  end
-  flow do
-    para "As a circle"
-    stroke "blue"
-    fill "pink"
-    oval 30, 30, 80, center: true
-  end
-  flow do
-    para "Keyword arguments:"
-    fill "green"
-    oval top: 20, left: 20, height: 160, width: 90, center: true, stroke: "red", strokewidth: 4
+    stroke green
```

Comment:
I've been trying to un-quote properties like stroke here more. Scarpe doesn't mind it being a string, but Shoes3 chokes on it -- tries to load it as a file. No reason we can't support both in Scarpe, but I like being able to load side by side and compare.